srcpkgs/mpvpaper/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'mpvpaper'
+pkgname=mpvpaper
+version=1.3
+revision=1
+build_style=meson
+hostmakedepends="pkg-config wayland-devel"
+makedepends="wayland-protocols wlroots-devel mpv-devel"
+depends="wayland-protocols"
```

Comment:
Does this actually depend on `wayland-protocols` at runtime?

Reply:
My bad, it doesn't. Fixed it.